lib/Echidna/Types/Tx.hs

```diff
@@ -109,3 +112,6 @@ getResult (VMFailure InvalidMemoryAccess)       = ErrorInvalidMemoryAccess
 getResult (VMFailure CallDepthLimitReached)     = ErrorCallDepthLimitReached
 getResult (VMFailure (MaxCodeSizeExceeded _ _)) = ErrorMaxCodeSizeExceeded
 getResult (VMFailure PrecompileFailure)         = ErrorPrecompileFailure
+getResult (VMFailure UnexpectedSymbolicArg)     = ErrorUnexpectedSymbolic
+getResult (VMFailure DeadPath)                  = ErrorDeadPath
+getResult (VMFailure (Choose _))                = ErrorChoose -- not entirely sure what this is
```

Comment:
https://github.com/dapphub/dapptools/blob/786f8cac6450de29ddc92ab59a5056fad44abee0/src/hevm/src/EVM.hs#L1537-L1538

Reply:
That's if you use symbolic execution. I will remove that comment.